File: kata/C#/8KYU/Basic Math.cs
namespace Solution
{
  public static class Program
  {
    public static double basicOp(char op, double val1, double val2)
    {
    	switch(op.ToString())
      {
      	case "+":
         return val1 + val2;
      	case "-":
         return val1 - val2;
      	case "*":
         return val1 * val2;
      	case "/":
         return val1 / val2;
       }
       return -1;      
    }
  }
}

File: kata/PHP/8KYU/Basic Math.php
function basicOp($op, $val1, $val2)
{
  switch($op){
    case '+':
    	$result = $val1 + $val2;
    	break;
    case '-':
    	$result = $val1 - $val2;
    	break;
    case '*':
    	$result = $val1 * $val2;
    	break;
    case '/':
    	$result = $val1 / $val2;
    	break;
    }
    return $result;
 
}